client/src/lib.rs

```diff
@@ -13,7 +13,7 @@ pub struct HpAdminKeypair(Keypair);
 struct Payload {
     method: String,
     request: String,
-    body: String,
+    body: Option<String>,
```

Comment:
Does HTTP have a distinction between missing body and empty body (`body == ""`)?

Reply:
Empty body is a special case and depends on the implementation of the code. In the client empty body passed to axios is simply `undefined`. `undefined` translates to JsValue `None`. Hyper in Rust reads body as `hyper::Chunk` convertible to  `Vec<u8>`, that I convert to `String` for signature verification. I added edge case check that returns None variant for empty body.